seqr/views/apis/phenotips_api.py

```diff
@@ -507,5 +511,26 @@ def _get_phenotips_username_and_password(user, project, permissions_level):
     return auth_tuple
 
 
+def sync_phenotips_data(project, individual):
+    """Retrieve and update the phenotips_data and phenotips_patient_id fields for the given Individual
+
+    Args:
+        project (Model): Project model
+        individual (Model): Individual model
+    """
+
+    if not phenotips_patient_exists(individual):
+        return False
+
+    data_json = _get_patient_data(
+        project,
+        individual,
+    )
+
+    _update_individual_phenotips_data(individual, data_json)
```

Comment:
Added this because I found some records don't have `phenotips_patient_id` but do have `phenotips_eid`.

Reply:
So then instead of this broadly named `sync_phenotips_data` function why not make it `set_phenotips_id` or `ensure_phenotips_id` which checks for that case and only calls update if they don't have `phenotips_patient_id`

Reply:
update_projects_in_new_schema also uses it, and needs the full functionality. 
In principle, the data should always be in sync, so might be better to just get rid of this of this method and call the methods it calls directly when needed (eg. in update_projects_in_new_schema)?

Reply:
agreed

Reply:
done making these changes